django/middleware/security.py

```diff
@@ -43,4 +44,7 @@ def process_response(self, request, response):
         if self.xss_filter:
             response.setdefault('X-XSS-Protection', '1; mode=block')
 
+        if self.referrer_policy is not None:
+            response.setdefault('referrer-policy', self.referrer_policy)
```

Comment:
It should be possible to specify multiple values to allow for fallback where a value is not supported by a user agent: https://www.w3.org/TR/referrer-policy/#policy-token